docs/runtime-contract.md

```diff
@@ -411,13 +411,13 @@ MUST be provided:
 | `/var/log` | MUST be a directory with write permissions for logs storage. Implementations MAY permit the creation of additional subdirectories and log rotation and renaming. |
 | `/dev/log` | MUST be a writable socket to syslog                                                                                                                              |
 
-In addition, the following files may be overridden by the runtime environment to
+In addition, the following files MAY be overridden by the runtime environment to
```

Comment:
`/etc/resolve.conf` is listed as "SHOULD be set". It is somewhat confusing to have the MAY paragraph prefix this table. 

I would opt to change the word 'may' in the leading paragraph and rely on the Description to specify the keywords.

Reply:
I was a bit bothered too as I was tweaking this... I'll do that.